recipes/di/all/conanfile.py

```diff
@@ -0,0 +1,65 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class DiConan(ConanFile):
+    name = "di"
+    license = "BSL-1.0"
+    homepage = "https://github.com/boost-experimental/di"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "[Boost].DI: C++14 Dependency Injection Library."
+    topics = ("dependency-injection", "metaprogramming", "design-patterns")
+    exports_sources = ["patches/**"]
+    settings = ("compiler",)
```

Comment:
this should just work
```suggestion
    settings = "compiler"
```

Reply:
I've looked at other recipes and it's the most used way, you're right.